### PR TITLE
32 create pagination urls for list of hotels and rooms close #32

### LINKED ISSUE #32: Create pagination urls for list of hotels and rooms
Please add the pagination functionality using rest api when fetching list of hotels and rooms.

### PR DESCRIPTION
**Pull request by georgepoul**

### CODE REVIEW

File: src/test/java/com/sphy/hotelmanagementapplication/service/HotelServiceIT.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+        //given
+        int expected = 3;
+
+        //when
+        List<HotelDTO> hotelDTOS = hotelService.getHotels(0,expected,"id");
+
+        //then
+        assertEquals(expected, hotelDTOS.size());
+
```
Comment (DimK10) [inline src/test/java/com/sphy/hotelmanagementapplication/service/HotelServiceIT.java:60]: Please add a check in some of the fields in one of the hotelDTO's returned. This test confirms that the elements returned as faras the list size is ok, but we should do a basic check on those elements

File: src/test/java/com/sphy/hotelmanagementapplication/service/HotelServiceTest.java [hunk 2]
```diff
@@ -192,108 +186,6 @@ void saveHotels() {
             // todo
         }
 
-        @Test
-        void getHotels() throws Exception {
```
Comment (DimK10) [inline src/test/java/com/sphy/hotelmanagementapplication/service/HotelServiceTest.java:196]: why did you remove those tests?

File: src/test/java/com/sphy/hotelmanagementapplication/service/HotelServiceTest.java [hunk 3]
```diff
@@ -413,4 +305,17 @@ void updateHotel() {
             // todo
         }
 
+        @Test
+        void getHotels(){
```
Comment (DimK10) [inline src/test/java/com/sphy/hotelmanagementapplication/service/HotelServiceTest.java]: This is not countHotels

File: src/test/java/com/sphy/hotelmanagementapplication/service/RoomServiceIT.java [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        void getHotels() throws Exception {
+
+            //given
+            int expected = 6;
+
+            //when
+            List<RoomDTO> hotelDTOS = roomService.getRooms(0,expected,"id");
+
+            //then
+            assertEquals(expected, hotelDTOS.size());
```
Comment (DimK10) [inline src/test/java/com/sphy/hotelmanagementapplication/service/RoomServiceIT.java]: The same as HotelServiceIT comment

File: src/test/java/com/sphy/hotelmanagementapplication/service/RoomServiceTest.java [hunk 5]
```diff
@@ -100,22 +97,6 @@ void saveRooms() {
 		// todo
 	}
 
-	@Test
-	void getRooms() throws Exception {
```
Comment (DimK10) [inline src/test/java/com/sphy/hotelmanagementapplication/service/RoomServiceTest.java:104]: Again, this test is for all rooms not their count why did you remove it

**Review by DimK10 — CHANGES_REQUESTED** [review]
Please check the comments

File: src/test/java/com/sphy/hotelmanagementapplication/service/HotelServiceTest.java (hunk 2 above)
Comment (georgepoul) [reply src/test/java/com/sphy/hotelmanagementapplication/service/HotelServiceTest.java:196]: I have made an IT test for that.

File: src/test/java/com/sphy/hotelmanagementapplication/service/RoomServiceTest.java (hunk 5 above)
Comment (georgepoul) [reply src/test/java/com/sphy/hotelmanagementapplication/service/RoomServiceTest.java:104]: I have made an IT test for that.

**Review by DimK10 — APPROVED** [review]

**Pull request merged**